frontend/src/views/main/project/InspectorWrapper.vue

```diff
@@ -197,7 +197,7 @@ const rowIdxInPage = ref(0);
 const regressionThreshold = ref(0.1);
 const percentRegressionUnit = ref(true);
 
-const canPrevious = computed(() => !shuffleMode.value && rowNb.value > 0);
+const canPrevious = computed(() => rowNb.value > 0);
```

Comment:
@andreybavt it's already doing that, because the data points come from backend already shuffled. To get the exactly previous data point of the original order we could use the property `_GISKARD_INDEX_` instead of `rowNb`

Reply:
So, to wrap up, the behaviour of `rowNb.value -= 1` (current behaviour) is to go back to the last data point seen instead of the original index - 1

Reply:
got it thanks